### PR TITLE
pnpm: refresh page

### PR DESCRIPTION
**Pull request by leonfeng**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 7.x**

Thank you for reviewing my pull request. I will comment on the diffs with justifications.

### CODE REVIEW

File: pages/common/pnpm.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 > Manage Node.js projects and their module dependencies.
 > More information: <https://pnpm.io>.
 
-- Interactively create a `package.json` file:
+- Create a `package.json` file:
```
Comment (leonfeng) [inline pages/common/pnpm.md:7]: `pnpm init` is no longer interactive.

File: pages/common/pnpm.md [hunk 2]
```diff
@@ -14,19 +14,19 @@
 
 - Download a specific version of a package and add it to the list of dependencies in `package.json`:
 
-`pnpm install {{module_name}}@{{version}}`
+`pnpm add {{module_name}}@{{version}}`
 
-- Download a package and add it to the list of dev dependencies in `package.json`:
+- Download a package and add it to the list of [D]ev dependencies in `package.json`:
```
Comment (leonfeng) [inline pages/common/pnpm.md:19]: I'm aware that long parameters are encouraged. But I doubt anyone would prefer to type `--save-dev` than `-D`.

File: pages/common/pnpm.md [hunk 3]
```diff
@@ -14,19 +14,19 @@
...
-- Download a package and install it globally:
+- Download a package and install it [g]lobally:
 
-`pnpm install -g {{module_name}}`
+`pnpm add -g {{module_name}}`
 
 - Uninstall a package and remove it from the list of dependencies in `package.json`:
 
-`pnpm uninstall {{module_name}}`
+`pnpm remove {{module_name}}`
```
Comment (leonfeng) [inline pages/common/pnpm.md:29]: Although `uninstall` is a supported alias, the official subcommand is `remove`.

**Review by CleanMachine1 — APPROVED** [review]
lgtm, thanks for your contribution

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution @leonfeng

**Pull request merged**